csrc/scheduler/utils.cpp

```diff
@@ -1266,10 +1252,14 @@ void FindAllMappedDims::propagateSibling(TensorView* from, TensorView* to) {
 std::unordered_set<IterDomain*> FindAllMappedDims::get() const {
   std::unordered_set<IterDomain*> mapped_id_set;
   for (auto entry : mapped_root_ids_) {
-    mapped_id_set.emplace(entry.second);
+    if (entry.second != nullptr) {
+      mapped_id_set.emplace(entry.second);
+    }
   }
   for (auto entry : mapped_rfactor_ids_) {
-    mapped_id_set.emplace(entry.second);
+    if (entry.second != nullptr) {
+      mapped_id_set.emplace(entry.second);
+    }
```

Comment:
There should be no behavior change here, I am just making sure that `mapped_id_set` does not contain `nullptr` so we can just use `ir_utils::toString` to print it.

Reply:
Why can it be `nullptr`? Can we just ignore it?

Reply:
`nullptr` mean in the given tensor, nothing is mapped. With the current code organization, it is just slightly more convenient to put a `nullptr` to `mapped_rfactor_ids_` rather than not putting anything.